spec/file_manipulation_spec.rb

```diff
@@ -17,7 +17,7 @@
   describe '#info_log_to_json' do
     it 'tests if the method is returning correctly the JSON object' do
       file = FileManipulation.new('false_games.log')
-      json = { 'false_games.log' => { 'lines' => 3, 'players' => ['Isgalamido'] } }.to_json
+      json = { 'false_games.log' => { 'lines' => 11, 'players' => ['Isgalamido', 'Dono da Bola', 'Assasinu Credi'], 'kills' => { 'Isgalamido' => -1, 'Dono da Bola' => 1, 'Assasinu Credi' => 0 }, 'total_kills' => 6 } }.to_json
```

Comment:
Quebra essas linhas qnd fica mto grande assim, ta usando [rubocop](https://github.com/rubocop/rubocop) ai? É uma boa pratica configurar um linter pra ajudar nessa parte :D

```suggestion
      json = { 
        'false_games.log' => {
          'lines' => 11,
          'players' => ['Isgalamido', 'Dono da Bola', 'Assasinu Credi'],
          'kills' => { 'Isgalamido' => -1, 'Dono da Bola' => 1, 'Assasinu Credi' => 0 },
          'total_kills' => 6
        }
      }.to_json
```